{"author":{"id":"4d85ccdde3ad47c902de21b580354b67ed0cb07caa04e32d04ea93112f7f027e"},"ops":[{"type":3,"timestamp":1745883038,"nonce":"7TT+Xt3grgYnHmFdAls2dxd5Z00=","metadata":{"github-id":"IC_kwDOCGKeh86pGUUu","github-url":"https://github.com/git-bug/git-bug/issues/749#issuecomment-2837005614"},"message":"As a stop-gap to solve this and any related issues (i'll link any i find in this comment later; i remember seeing this pop up a few times), an \"infinite\" incremental backoff can be implemented for all operations that pull from a bridge. Whether or not this is actually infinite, or if it kills the process after a certain number of retries is TBD. I think it would be reasonable to exit if we get a known response back that equates to \"you have reached the rate limit\".","files":null},{"type":2,"timestamp":1745883058,"nonce":"/2JhiJH6H+Q2fyiqnbbC6eaVBQU=","metadata":{"github-id":"RTE_lADOCGKeh85EUlaozwAAAAQPxAQ0"},"title":"Rate limit reached when pulling from a bridge","was":"problems importing from github.com/nixos/nixpkgs"},{"type":5,"timestamp":1746759877,"nonce":"+HbjVZqThVdnmNQKjwv2Sky3m5s=","metadata":{"github-id":"UNLE_lADOCGKeh85EUlaozwAAAAQYEomN"},"added":[],"removed":["lifecycle/pinned"]}]}